internal/xds/resolver/xds_resolver.go

```diff
@@ -230,8 +230,12 @@ type xdsResolver struct {
 	curConfigSelector stoppableConfigSelector
 }
 
-// ResolveNow is a no-op at this point.
-func (*xdsResolver) ResolveNow(resolver.ResolveNowOptions) {}
+// ResolveNow calls ResolveNow on the dependency manager.
```

Comment:
This comment needs to be updated because the method is no longer called `ResolveNow` on the dependency manager.